convert2rhel/subscription.py

```diff
@@ -489,7 +465,19 @@ def __call__(self):
                     )
 
                 else:
-                    loggerinst.info("Registering via org/activation_key: %s" % " ".join(utils.hide_secrets(self.args)))
+                    loggerinst.info(
+                        "Registering via org/activation_key: "
+                        "Organization: %s\nActivation Key:%s\nRegister Options: %s\n"
+                        "Connection Options: %s\nLocale Setting: %s\n"
+                        % (
+                            self.org,
+                            "*" * 5,
+                            REGISTER_OPTS_DICT,
+                            self.connection_opts,
+                            i18n.SUBSCRIPTION_MANAGER_LOCALE,
+                        )
+                    )
```

Comment:
@pr-watson, I would modify this logging a bit and make it a bit simpler to read and to maintain later, maybe something like this: 

 ```suggestion
                    loggerinst.info("Organization: %s", self.org)
                    loggerinst.info("Activation Key: %s", "*" * 5)
                    loggerinst.info("Connection Options: %s", self.connection_opts)
                    loggerinst.info("Locale settings: %s", i18n.SUBSCRIPTION_MANAGER_LOCALE)
```

I would say it's totally fine to have multiple log calls as far as they are easier to ready and maintain... What do you think? 

NOTE: This also applies to the logging text above that you have in the same way.